dashboard/app/controllers/concerns/pd/workshop_filters.rb

```diff
@@ -0,0 +1,46 @@
+module Pd::WorkshopFilters
+  extend ActiveSupport::Concern
```

Comment:
how about a brief top-level comment explaining what this file does, unless that violates some convention?